test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java

```diff
@@ -1075,8 +1099,13 @@ private void recreateNode(final Settings newSettings, final Runnable onTransport
                 .put(newSettings)
                 .put(NodeEnvironment.NODE_ID_SEED_SETTING.getKey(), newIdSeed)
                 .build();
-            Collection<Class<? extends Plugin>> plugins = node.getClasspathPlugins();
-            node = new MockNode(finalSettings, plugins);
+            Collection<PluginInfo> plugins = node.getClasspathPlugins();
+            node = new MockNode(
+                finalSettings,
+                plugins,
```

Comment:
Do we need `Collection<PluginInfo> plugins = `?"

```suggestion
                node.getClasspathPlugins();,
```

Reply:
Applied this change.